flight/PiOS/STM32F10x/pios_reset.c

```diff
@@ -0,0 +1,48 @@
+/**
+ ******************************************************************************
+ * @addtogroup PIOS PIOS Core hardware abstraction layer
+ * @{
+ * @addtogroup PIOS_RESET Reset functions
+ * @brief Hardware functions to deal with the reset register
+ * @{
+ *
+ * @file       pios_reset.c
+ * @author     Kenn Sebesta, Copyright (C) 2015.
```

Comment:
what are your thoughts on updating this to also include taulabs like other files? not sure if there are implications of this are but generally we haven't used individual names.